Cache running in test mode
Traceback (most recent call last):
  File "/home/r_r/http_server/httpserver", line 24, in <module>
    repli_cache = RepliCache(test_mode=cache_test_mode)
  File "/home/r_r/http_server/cache.py", line 41, in __init__
    self.build()
  File "/home/r_r/http_server/cache.py", line 53, in build
    with open("pageviews.csv") as article_file:
FileNotFoundError: [Errno 2] No such file or directory: 'pageviews.csv'
